src/libraries/System.Private.CoreLib/src/Internal/AssemblyAttributes.cs

```diff
@@ -22,5 +22,6 @@
 
 [assembly: AssemblyMetadata("Serviceable", "True")]
 [assembly: AssemblyMetadata(".NETFrameworkAssembly", "")]
```

Comment:
Completely unrelated to this PR, but it caught my eye... is this value still desired?  Should the "Framework" piece be removed?
cc: @ericstj

Reply:
Certain things look for this string. ex.

https://github.com/dotnet/corert/blob/24f1edc938af9954eac9a3385958787ca64956d5/src/ILCompiler.Compiler/src/Compiler/FrameworkStringResourceBlockingPolicy.cs#L58-L87